source/shared/core_results.cpp

```diff
@@ -45,7 +45,7 @@ namespace {
 
 // *** internal constants ***
 
-const int INITIAL_FIELD_STRING_LEN = 256;    // base allocation size when retrieving a string field
+const int INITIAL_FIELD_STRING_LEN = 1024;      // base allocation size when retrieving a string field
```

Comment:
can you only increase the initial buffer size when column encryption is enabled? Also, if column encryption is enabled, I would love to blast the initial buffer size to 8000 (max size for varchar(n))

Reply:
I think it's reasonable to simply use 8000 regardless of AE setting. We use this buffer size for varchar / char parameters with unknown size.